Description: these tests requires some git sub-directories
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-05-03

--- a/tests/json-test-suite.ts
+++ b/tests/json-test-suite.ts
@@ -49,6 +49,7 @@
   })
 }
 
+/*
 describe('json-test-suite test_parsing', () => {
   const dir = resolve(__dirname, 'json-test-suite', 'test_parsing')
   for (const fn of readdirSync(dir)) {
@@ -68,3 +69,4 @@
     testSuccess(basename(fn, '.json'), src)
   }
 })
+*/
